dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServerFactory.java

```diff
@@ -440,9 +440,17 @@ private static void addAppsToVerificationList(RemoteFile[] files, LibertyServer
     }
 
     public static void recursivelyCopyDirectory(Machine machine, LocalFile localDirectory, RemoteFile destination) throws Exception {
+        recursivelyCopyDirectory(machine, localDirectory, destination, false);
+    }
+
+    public static void recursivelyCopyDirectory(Machine machine, LocalFile localDirectory, RemoteFile destination, boolean cleanup) throws Exception {
```

Comment:
perhaps rename the `cleanup` boolean to something a bit more obvious such as `mergeFolders` (default=true)